examples/python/Calibration/calibration_dump_sockets.py

```
@@ -18,15 +18,15 @@
 
 for cam in camera_sockets:
     print(f'\n========== {cam.name} ==========')
-    hasEeprom = device.isEepromAvailable(cam)
+    hasEeprom = device.isCBAEepromAvailable(cam)
     print(f'[{cam.name}] Is EEPROM available: {hasEeprom}')
     if hasEeprom:
         try:
-            print(f'[{cam.name}] User calibration: {json.dumps(device.readCalibration2(cam).eepromToJson(), indent=2)}')
+            print(f'[{cam.name}] User calibration: {json.dumps(device.readCBACalibration2(cam).eepromToJson(), indent=2)}')
         except Exception as ex:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)